Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1851.cs

```diff
@@ -1,26 +1,31 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
 using Xamarin.Forms.CustomAttributes;
 using Xamarin.Forms.Internals;
 
+#if UITEST
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
 namespace Xamarin.Forms.Controls.TestCasesPages
 {
+#if UITEST
+	[Category(UITestCategories.ListView)]
+#endif
 	[Preserve (AllMembers=true)]
 	[Issue (IssueTracker.Github, 1851, "ObservableCollection in ListView gets Index out of range when removing item", PlatformAffected.Android)]
 	public class Issue1851 : ContentPage
```

Comment:
@adrianknight89  changing the base class to *TestContentPage* and moving the code to OnInit should fix the build errors

Reply:
Done. Another thing I noticed is you have 18 issue tests under `Xamarin.Forms.Controls.TestCasesPages` whereas most everything belongs to the `Xamarin.Forms.Controls.Issues` namespace.

Reply:
@StephaneDelcroix please verify the new commit. Thanks!